CHANGELOG.md

```diff
@@ -59,6 +60,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
   crud.SelectRequest (#320)
 - Incorrect options (`vshard_router`, `fields`, `bucket_id`, `mode`,
   `prefer_replica`, `balance`) setup for crud.GetRequest (#335)
+- Tests with crud 1.4.0 (#336)
```

Comment:
Let's better write what we are support 1.4.0 now than tests are fixed.

Reply:
But master already supports crud 1.4.0, everything work fine, the only issue is the test case on error

Reply:
There is an entry about `crud.schema` in a separate commit
https://github.com/tarantool/go-tarantool/pull/336/commits/7a70686bb2c83cb91379845f6912d49d15dcce15